subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -34,6 +34,53 @@ config BT_ATT_TX_MAX
 	  amount the calls will block until an existing queued PDU gets
 	  sent.
 
+config BT_EATT
+	bool "Enhanced ATT Bearers support"
+	default y
+	depends on BT_L2CAP_DYNAMIC_CHANNEL
+	help
+	  This option enables support for Enhanced ATT bearers support. When
+	  enabled additional L2CAP channels can be connected as bearers enabling
+	  multiple outstanding request.
+
+config BT_EATT_MAX
+	int "Maximum number of Enhanced ATT bearers"
+	default 3
+	range 1 16
+	depends on BT_EATT
+	help
+	  Number of Enhanced ATT bearers available.
+
+config BT_EATT_RX_MTU
+	int "Maximum supported Enhanced ATT MTU for incoming data"
+	default 70
+	range 70 519
+	depends on BT_EATT
+	help
+	  Maximum size incoming PDUs on EATT bearers. This option influences the
+	  stack buffer size and by that may also limit the outgoing MTU.
+
+config BT_EATT_SEC_LEVEL
+	int "Enhanced ATT bearer security level"
+	default 2
+	range 1 4
+	depends on BT_EATT
+	help
+	  Security level of EATT bearers:
+	  Level 1 (BT_SECURITY_LOW) = No encryption or authentication required
+	  Level 2 (BT_SECURITY_MEDIUM) = Only encryption required
+	  Level 3 (BT_SECURITY_HIGH) = Encryption and authentication required
+	  Level 4 (BT_SECURITY_FIPS) = Secure connection required
```

Comment:
We deprecated the LOW/MEDIUM/HIGH/FIPS defines, so I don't think they should be mentioned here (they're only called L1, L2, etc.

Reply:
Right, will update it.